library/commandline/src/modules/CommandLine.rb

```diff
@@ -488,10 +488,10 @@ def Parse(arguments)
                 Ops.set(givenoptions, o, i)
               end
             else
-              if v == ""
-                ret = false
+              ret = if v == ""
+                false
               else
-                ret = TypeRepository.is_a(v, opttype)
+                TypeRepository.is_a(v, opttype)
               end
```

Comment:
I worry this is not much readable, but later conversion helps more with ternary operator.